app/src/main/java/com/example/mycalculator/mvp/CalculatorPresenter.java

```diff
@@ -0,0 +1,18 @@
+package com.example.mycalculator.mvp;
```

Comment:
yo can create a package called presenter, and put this class inside

Reply:
Done. Thx!